cmake/jsoncpp.cmake

```diff
@@ -6,15 +6,17 @@ else()
     set(JSONCPP_CMAKE_COMMAND ${CMAKE_COMMAND})
 endif()
 
-# Disable implicit fallthrough warning in jsoncpp for gcc >= 7 until the upstream handles it properly
-if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 7.0)
-    set(JSONCCP_EXTRA_FLAGS -Wno-implicit-fallthrough)
+get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
```

Comment:
I think there is easier way to do this: https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html.

```
include(GNUInstallDirs)

set(JSONCPP_LIBRARY "${prefix}/${CMAKE_INSTALL_LIBDIR}/...
```

Reply:
Nice! Thats much better!